Manito/Manito/Screens/CreateNickName/CreateNickNameViewController.swift

```diff
@@ -64,9 +73,43 @@ class CreateNickNameViewController: BaseViewController {
         }
     }
     
+    // MARK: - Seletors
+    
+    @objc func didTapDoneButton() {
+        if let text = roomsNameTextField.text, !text.isEmpty {
+            nickName = text
+        }
+        roomsNameTextField.resignFirstResponder()
+    }
+    
+    // MARK: - Funtions
+    
+    func setupDelegation() {
+        roomsNameTextField.delegate = self
+    }
+    
     // MARK: - Configure
     
     override func configUI() {
         super.configUI()
     }
 }
+
+// MARK: - Extension
+extension CreateNickNameViewController : UITextFieldDelegate {
+    
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        roomsNameTextField.resignFirstResponder()
+        return true
+    }
+    
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        if range.location == 0 && range.length != 0 {
+            doneButton.isDisabled = true
+        }
+        else {
+            doneButton.isDisabled = false
+        }
```

Comment:
p5;

```suggestion
        doneButton.isDisabled = !textField.hasText()
```

요렇게 써도 같은 의미이지 않을까요?

Reply:
오,.. 짱